vpn.py

```diff
@@ -22,7 +22,10 @@ def parse_args():
     parser.add_argument('--create-hma-configs', dest='create_HMA',
                         action="store_true",
                         help='Create the openvpn config files for HMA')
-    parser.add_argument('--log-file', dest='log_file', default='vpn-log.log',
+    parser.add_argument('--exclude', "-e", dest='exclude_list', default=None,
```

Comment:
what is the use case for this feature? When would we need to exclude VPNs from a certain country?